### PR TITLE
Get/api/users

### PR DESCRIPTION
**Pull request by AdilBulsari**

### CODE REVIEW

File: controller/getTopic.controller.js [hunk 1]
```diff
@@ -1,7 +1,6 @@
 const { fetchTopic } = require("../model/fetchTopic.model");
 
 exports.getTopic = (req, res) => {
-  console.log("in get toppic controller");
   fetchTopic().then((data) => {
```
Comment (douglashellowell) [inline controller/getTopic.controller.js]: try to get into the habit of naming your parameters clear names

`data` isn't very clear - `topics` would be better here

(have a look at where you've used `data` elsewhere too!)

File: __tests__/app.test.js [hunk 2]
```diff
@@ -53,6 +54,7 @@ describe.only("GET /api/articles/:article_id", () => {
     return request(app)
       .get("/api/articles/12")
       .expect(200)
+
       .then(({ body: { topic } }) => {
```
Comment (douglashellowell) [inline __tests__/app.test.js]: this is coming back on the key of topic - it should be an article!

**Comment by douglashellowell** [comment]
Great stuff so far!

There are a few places where you should tidy up your code (make sure you are running Prettier before commiting your code!)

Make sure you have not got any `.only`s on your tests!
All your tests need to be passing before you push your code

It will be easier if you fully error handle an endpoint before moving on

Your `GET /api/articles/:article_id` should test the following
```
/api/articles/999999
/api/articles/not-an-id
```

Your `PATCH /api/articles/:article_id` endpoint only has one `200` test. You should cover the following:
```
/api/articles/999999 
/api/articles/not-an-id

/api/articles/1  => { inc_votes: 'not a number' }
/api/articles/1 =>  {  }
```

**Comment by douglashellowell** [comment]
Whereever you have a `forEach` in your tests make sure you are checking that the array is not coming back empty!

A `forEach` used on an empty array will not run it's function and your tests will always pass (even if your code is broken!)

You should check the array has a `length` greater than 0 before your `forEach`

**Comment by douglashellowell** [comment]
you should delete `test.sql` and `output.txt` from your repos

if you want to make them again you should add them to the `.gitignore` so they don't get put on github

**Review by HeatherEver — COMMENTED** [review]
Looks good Adil!

**Pull request merged**